ci/aws-lambda-ruby-dev/3.2/Dockerfile

```diff
@@ -1,19 +1,25 @@
-FROM public.ecr.aws/lambda/ruby:3.2
-
-# Install tools used when installing / building gems
-RUN yum install -y postgresql-devel sqlite-devel openssl-devel wget make gcc gcc-c++ git \
- && yum clean all \
- && rm -rf /var/cache/yum
+FROM amazonlinux:2023
```

Comment:
can we use fully qualified here: eg docker.io/amazonlinux:2023
Makes it a little clearer but also works with podman  and other container engines. :)

Reply:
Fixed